packages/packages/api/src/medical/models/common/address.ts

```diff
@@ -2,12 +2,12 @@ import { z } from "zod";
 import { usStateSchema } from "./us-data";
 
 export const addressSchema = z.object({
-  addressLine1: z.string().min(1),
+  addressLine1: z.string().optional(),
   addressLine2: z.string().optional().nullable(),
-  city: z.string().min(1),
-  state: usStateSchema,
-  zip: z.string().length(5),
-  country: z.string().min(1),
+  city: z.string().optional(),
+  state: usStateSchema.optional(),
+  zip: z.string(),
+  country: z.string().optional(),
```

Comment:
Few things to consider here:

- We're using `addressSchema` for orgs/facilities as well, for which those fields should not be optional (ie they're required to make an org in CW).
- Will this break our form validation for patients?
- If making this change to our models, we'll need to update the docs accordingly - ie we use [this snippet](https://github.com/metriport/metriport/blob/ed3f188aa9a463eda21a379c34f1279008ac29cc/docs/_snippets/address.mdx#L1-L26) in our docs.

Reply:
Perhaps we should just fix the sandbox payload for now, and tackle the optional field change in a separate issue?